UnityProject/Assets/Scripts/Objects/Pipes/Devices/AirVent.cs

```diff
@@ -11,7 +12,7 @@ public class AirVent : MonoPipe
 		public bool SelfSufficient = false;
 
 		// minimum pressure needs to be a little lower because of floating point inaccuracies
-		public float MaxTransferMoles = 100;
+		public float MaxTransferMoles = 10;
```

Comment:
remember to update prefabs, also this will make scrubbers slow as balls when trying to get rid of plasma, to be fair there already pretty slow